packages/http-client-csharp/generator/TestProjects/CadlRanch/http/type/scalar/tspCodeModel.json

```diff
@@ -127,7 +127,7 @@
         "CrossLanguageDefinitionId": "TypeSpec.string",
         "Decorators": []
        },
-       "BodyMediaType": "Json",
+       "BodyMediaType": "Text",
```

Comment:
This is the regression after we remove hard-coded `Json` value. From the `ContentTypes` below, I think it should be `Json`. But luckily the generated codes are not impacted.